static/js/containers/DashboardPage.js

```diff
@@ -550,6 +550,10 @@ class DashboardPage extends React.Component {
     );
   }
 
+  openLinkInNewTab = (url: ?string): void => {
+    window.open(url, '_blank');
+  }
```

Comment:
i think it is not possible, i am asserting the method in dashboard is called with correct params